extensions/sql-migration/src/dialog/skuRecommendationResults/GenerateProvisioningScriptDialog.ts

```diff
@@ -135,6 +135,26 @@ export class GenerateProvisioningScriptDialog {
 			await selectAzureAccountDialog.initialize();
 		});
 
+		const copyToClipboardButton = _view.modelBuilder.button()
+			.withProps({
+				buttonType: azdata.ButtonType.Normal,
+				label: constants.COPY_TO_CLIPBOARD,
+				width: 130,
+				height: 36,
+				iconHeight: 16,
+				iconWidth: 16,
+				iconPath: IconPathHelper.copy,
+				CSSStyles: {
+					...styles.TOOLBAR_CSS
+				}
+			}).component();
+
+		copyToClipboardButton.onDidClick(async () => {
+			if (this.model._armTemplateResult.templates) {
+				void vscode.env.clipboard.writeText(this.model._armTemplateResult.templates[0]);
```

Comment:
if templates are null by any means than this will break.

Reply:
I don't think it will break , its just the copy action will do nothing .